client/components/Registration-Page/register.mjs

```diff
@@ -0,0 +1,31 @@
+import { inputListener } from '../input/input.mjs';
+
+inputListener('registerFormInput', 'input', checkInputValidation);
+function checkInputValidation() {
+   const inputs = document.getElementsByClassName('registerFormInput');
+   let validate = true;
+   for (let item of inputs) {
+      if (item.id === 'password1') {
+         checkPasswords(item);
+      }
+      if (!item.checkValidity()) {
+         validate = false;
+      }
+   }
+   (validate) ? document.getElementById('createAccount').disabled = false : document.getElementById('createAccount').disabled = true;
+}
+
+function checkPasswords(item) {
+   const passwordVarification = document.getElementById('password2');
+   (item.value != '') ? passwordVarification.setAttribute('pattern', item.value) : passwordVarification.setAttribute('pattern', '');
+}
+
+inputListener('inputCheckbox', 'input', showPassword);
+function showPassword() {
+   const inputPasswordFirst = document.getElementById("password1");
+   const inputPasswordSecond = document.getElementById("password2");
+   const Status = (input) => (input.type === 'password') ? 'text' : 'password';
```

Comment:
It's a function. Functions are typically written with small letter and name is usually a verb. So I would rename it to `getStatus`